OgreMain/src/OgreRenderTarget.cpp

```diff
@@ -487,9 +487,9 @@ namespace Ogre {
     //-----------------------------------------------------------------------
     String RenderTarget::writeContentsToTimestampedFile(const String& filenamePrefix, const String& filenameSuffix)
     {
-        struct tm *pTime;
-        time_t ctTime; time(&ctTime);
-        pTime = localtime( &ctTime );
+        std::time_t t { std::time(nullptr) };
+        auto pTime { std::localtime(&t) };
```

Comment:
```suggestion
        auto pTime = std::localtime(&t);
```

I know its up to taste, but I find this easier to read..